Automattic/jetpack
# VideoPress: Initial VideoUploadArea component

#### Changes proposed in this Pull Request:

Adds the `VideoUploadArea` component without upload functionality.

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Run `cd projects/js-packages/storybook && pnpm run storybook:dev`
* Confirm that all the stories of `Packages/Videopress/Video Upload Area` component work as expected

Desktop/tablet | Mobile
---|---
![Screenshot_2022-08-10_18-07-55-normal](https://user-images.githubusercontent.com/8486249/184021863-fed3edc1-7444-49d2-aba9-af5df6101240.png) | ![Screenshot_2022-08-10_18-09-06-normal-mobile](https://user-images.githubusercontent.com/8486249/184021904-12baacbe-0563-4ecd-9069-27035aad3486.png)

Uploading:
Desktop/tablet | Mobile
---|---
![Screenshot_2022-08-10_18-07-35-uploading](https://user-images.githubusercontent.com/8486249/184021979-bae7a3b8-7cc5-4941-9a9b-711846f20ebd.png) | ![Screenshot_2022-08-10_18-08-52-uploading-mobile](https://user-images.githubusercontent.com/8486249/184022036-c2524a2d-ed11-47e3-a10e-563868ec7462.png)

Hover:

https://user-images.githubusercontent.com/8486249/184022112-88c3cba4-fef8-4212-9772-a12dc1447d35.mp4



---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - 0-as-1202728110480523
